packages/sources/src/styling/global/dl.scss

```diff
@@ -177,6 +179,10 @@ dl {
         dd {
           break-inside: avoid-column;
         }
+
```

Comment:
dit mag wel samengepakt worden met `dd` hierboven, toch?

Reply:
deze snap ik niet, de selector hierboven is `dt, dd`?

Reply:
```
dd {
 break-inside: avoid-column;
 padding-left: 0;
}
```

Reply:
ah ja nu zie ik het, in mijn view was die dt weggevallen